packages/boot/index.js

```diff
@@ -0,0 +1,3 @@
+// @ts-check
+// Ambient types
+import '@agoric/vats/src/core/types.js';
```

Comment:
does something currently depend on this?

I don't think I completely understand what it's doing. But then... I'm not sure I want to.